### DISCUSSION TITLE
Read Aloud skipping endnote entries (by default)

### DISCUSSION THREAD

**Discussion by dazrand** [edrlab/thorium-reader#2635] [Ideas]
Read aloud skips any <li> item with the @epub:type='endnote' applied to it.  Stripping the attribute allows it to read those <li> entries.  In the sample below the second list item reads, the others are skipped. 

```<section epub:type="endnotes" role="doc-endnotes" id="ee">
<ol>
<li epub:type="endnote" id="r1">O’Toole, Michael. <cite>The Hermeneutic Spiral</cite>. New York: Routledge, 2018.</li>
<li id="r2">O’Toole, Michael. <cite>The Hermeneutic Spiral</cite>. New York: Routledge, 2018.</li>
<li epub:type="endnote" id="r3">O’Toole, Michael. <cite>The Hermeneutic Spiral</cite>. New York: Routledge, 2018.</li>
</ol>
</section>

**Comment by danielweck** [comment]
hello, have you tried disabling "skippability"?
(there's a checkbox in the extended playback section, i.e. the icon button cluster located in the center of the top bar)

**Comment by dazrand** [comment]
That did it. Should that be off be default?

**Comment by danielweck** [comment]
I converted the original issue into this GitHub discussion. This asks whether "skippability" should be activated by default when installing / first-time launching Thorium. Note that the boolean configuration switch applies to both authored EPUB3 Media Overlays (which includes DAiSY talking books) as well as Thorium's own TTS readaloud feature:
https://github.com/edrlab/thorium-reader/blob/fc74d55f927e3d2b433757c5fe4b504cf21e5ee9/src/common/redux/states/reader.ts#L56
